chaininfo/usecase/chain_info_usecase.go

```diff
@@ -20,28 +20,35 @@ type chainInfoUseCase struct {
 	// This has caused the healthcheck to pass with false positives in production.
 	// As a result, we need to keep track of the last seen height and time to ensure that the height is
 	// updated within a reasonable time frame.
-	lastSeenMx            sync.Mutex
-	lastSeenUpdatedHeight uint64
-	lastSeenUpdatedTime   time.Time
+	lastSeenMx          sync.Mutex
+	lastIngestedHeight  uint64
+	lastSeenUpdatedTime time.Time
 
 	priceUpdateHeightMx      sync.RWMutex
 	latestPricesUpdateHeight uint64
+
+	poolLiquidityUpdateHeightMx     sync.RWMutex
+	latestPoolLiquidityUpdateHeight uint64
 }
 
 // The max number of seconds allowed for there to be no updates
 // TODO: epoch???
 const (
 	MaxAllowedHeightUpdateTimeDeltaSecs = 30
 
-	// Number of heights of buffer between the latest state height and the latest price update height
-	// We fail the healtcheck if the difference between the two becomes greater than this constant.
-	priceUpdateHeightBuffer  = 50
-	initialPriceUpdateHeight = 0
+	// Number of heights of buffer between the latest state height and the latest price/pool liquidity update height
+	// We fail the healtcheck if the difference between the current and last becomes greater than this constant.
+	updateHeightBuffer  = 50
```

Comment:
Based on the comment, ```updateHeightBuffer``` seems to be used for threshold and alert purposes, shall we rename it something like ```updateHeightThreshold```? Or this is also used for some buffering purposes.